packages/flutter_hooks/test/use_search_controller_test.dart

```diff
@@ -71,6 +71,8 @@ void main() {
       controller.openView();
 
       expect(controller.isOpen, true);
+       // Advance fade animation duration.
+      await tester.pumpAndSettle(const Duration(seconds: 150));
```

Comment:
I don't like this change. I'm not testing animations here. Why would this be needed?